newIDE/app/src/ObjectsList/index.js

```diff
@@ -60,7 +60,7 @@ class ObjectsList extends Component<*, *> {
 
   render() {
     let { height, width, fullList, project, selectedObjectName } = this.props;
-
+    
```

Comment:
Nitpicking: can you remove these extra spaces/tabs?

Reply:
done